### PR TITLE
Upstream update: September 30

### PR DESCRIPTION
**Pull request by addaleax**
Skipped due to real conflicts (I’ll open a separate PR):
- nodejs/node@3a69ef55e06c
- nodejs/node@2b9b46cd5583
- nodejs/node@92d7e81b73ba

Skipped due to README conflicts:
- nodejs/node@93985ef83cf3
- nodejs/node@f30820fdca62

### CODE REVIEW

**Review by ghost — APPROVED** [review]

**Comment by BridgeAR** [comment]
@addaleax when you merge something here it can have impact on Node.js. One issue got closed because of this getting merged as there was a "Fixes xx" in a commit but that issue got reopened again and should stay open.

So it would be nice if you could try to find a different way of syncing with Node.js :smile:

**Comment by addaleax** [comment]
@BridgeAR Yeah, but unfortunately that issues gives a 500 Server Error for me rn? :/

> So it would be nice if you could try to find a different way of syncing with Node.js :smile:

It’s not just us that are having this problem – this happens from time to time with commits landing in libuv as well, for example. I guess we could try to check for those `Fixes:` tags or let somebody who doesn’t have commit access to `nodejs/node` merge these commits…

**Comment by BridgeAR** [comment]
Yeah, for me as well :/ (500)

I do not know a perfect solution for the tags either... letting people land without commit access is probably not always possible.
We might just have to always check the tags before landing something somewhere else?

**Pull request merged**
